package.json

```diff
@@ -13,7 +13,6 @@
     "build": "ng build on-prem",
     "build-cloud": "ng build cloud",
     "build-common": "ng build common",
-    "build-all": "ng build common --configuration production && ng build cloud --configuration production  && ng build on-prem --configuration production",
```

Comment:
Why was this removed? I found it helpful in preparing to build the docker containers.